app/src/main/java/no/usn/mob3000/ui/screens/auth/CreateUserScreen.kt

```diff
@@ -104,15 +108,15 @@ fun CreateUserScreen(
                             )
                         }
                     },
-                    colors = ButtonDefaults.buttonColors(DefaultButton),
+                    colors = ButtonDefaults.buttonColors(containerColor = MaterialTheme.colorScheme.primary),
                     modifier = Modifier.fillMaxWidth()
                 ) { Text(stringResource(R.string.auth_create_user_sign_up)) }
 
                 Spacer(modifier = Modifier.height(16.dp))
 
                 OutlinedButton(
                     onClick = onReturnToLoginClick
-                ) { Text(stringResource(R.string.auth_create_user_return_to_login)) }
+                ) { Text(stringResource(R.string.auth_create_user_return_to_login), color = MaterialTheme.colorScheme.onBackground) }
```

Comment:
Indentation. And [see this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847123849).